instrumentation/aws-sdk/aws-sdk-2.2/library/src/main/java/io/opentelemetry/instrumentation/awssdk/v2_2/TracingExecutionInterceptor.java

```diff
@@ -289,6 +305,40 @@ private void onSdkResponse(
     }
   }
 
+  @Override
+  public void afterTransmission(
+      Context.AfterTransmission context, ExecutionAttributes executionAttributes) {
+    if (!recordIndividualHttpError) {
+      return;
+    }
+
+    io.opentelemetry.context.Context otelContext = getContext(executionAttributes);
+    if (otelContext != null) {
+      Span span = Span.fromContext(otelContext);
+      SdkHttpResponse response = context.httpResponse();
+
+      if (!response.isSuccessful()) {
+        int errorCode = response.statusCode();
+        // we want to record the error message from http response
+        Optional<InputStream> responseBody = context.responseBody();
+        if (responseBody.isPresent()) {
+          try {
+            String errorMsg = IOUtils.toString(responseBody.get(), StandardCharsets.UTF_8);
```

Comment:
You don't need to add a dependency on `commons-io:commons-io`. Can you remain within boundaries of the standard library with the following?

```
 String errorMsg = new BufferedReader(new InputStreamReader(responseBody.get()))
   .lines().collect(Collectors.joining("\n"));
```

Reply:
Good suggestion!